packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -764,6 +764,10 @@ export class FluidDataStoreRuntime
 	 * @param outboundHandle - The handle of the outbound node that is referenced.
 	 */
 	private addedGCOutboundReference(srcHandle: IFluidHandle, outboundHandle: IFluidHandle) {
+		// Note: This is deprecated on IFluidDataStoreContext, and in an n/n-1 scenario where the
```

Comment:
Todo: Step through the unreferenced timestamp e2e tests in full compat mode to validate n/n-1 coverage